docs/create-a-business-component.md

```diff
@@ -0,0 +1,345 @@
+      />
+      <Marker position={[43.584296, 1.44182]}>
+        <Popup>
+          <span>My awesome store is HERE!</span>
+        </Popup>
+      </Marker>
+    </Map>
+  );
+};
+
+StoreLocator.propTypes = {};
+
+export default StoreLocator;
+```
+
+To allow import on the folder directly, we use those index files as proxies.
+There are not mandatory but your imports will be much cleaner with that.
+
+```js
+// src/theme/modules/StoreLocator/index.js;
+
+import StoreLocator from "./StoreLocator";
+
+export default StoreLocator;
+```
+
+### Enhancer
+
+Now we have to fetch our data from graphQL. For this we are going to create an
+**Enhancer**. it will be responsible to fetch and transform the store
+information to match our needs.
+
+```js
+// src/theme/modules/StoreLocator/EnhanceStoreLocator.js;
+
+import { graphql } from "react-apollo";
+
+export default StoreLocatorQuery =>
+  graphql(StoreLocatorQuery, {
+    props: ({ data }) => ({
+      loading: data.loading,
+      store: data.loading ? null : data.store
+    })
+  });
+```
+
+Here, we are using the `graphql` function which allow us to fetch data in our
+graphQL schema. It have two arguments, the first one is the **Query** we need to
+fetch (we will handle this part soon), in the second, we use the `props`
+property in the second parameter to compute the **Query** result `data` to a
+`loading` and a `store` properties.
+
+You can find all the available `graphql` options in the
+[**React Apollo** documentation](https://www.apollographql.com/docs/react/api/react-apollo.html)
+
+As you can see, our Enhancer need a **Query**, this a `.gql` (or `.graphql`)
+file that use the **GraphQL** syntax. In our case, it will look like:
+
+```gql
+// src/theme/modules/StoreLocator/StoreLocatorQuery.gql
+
+query StoreLocator {
+  store {
+    name
+    phone
+    owner {
+      email
+    }
+    coordinates {
+      longitude
+      latitude
+    }
+  }
+}
+```
+
+> You may think that some of those data are already fetched in our EnhanceHome
+> and thus, this is ineficient. But react-apollo will handle that for you and
+> will group the requests to make only the necessary requests.  
+> This allow to only think about what a component needs and responsibility
+> for the recovery of its data lies with him.
+
+### Making it dynamic
+
+Now that we have an Enhancer ready, we are going to use it in our StoreLocator.
+The major change here is the use of the data from the available props.
+
+When dealing with asynchronous resources like fetching data from the backend,
+you have to handle the loading state. Here we just show a simple `Loading..` to
+the user, but in a real life application, you would want to show placeholders or
+spinners.
+
+> You should also handle error cases, but it is not the purpose of this guide.
+> Just know that you could use
+> [Error Boundaries](https://reactjs.org/blog/2017/07/26/error-handling-in-react-16.html)
+> for that.
+
+```js
```

Comment:
If you have the time here, you could try to use diffs to illustrate the changes in a better way.

See https://gist.github.com/salmedina/ad8bea4f46de97ea132f71b0bca73663#file-markdowndiffexample-md for instance